MIGRATION.md

```diff
@@ -2664,6 +2666,132 @@ import { Tooltip } from '@contentful/f36-components';
 </Tooltip>;
 ```
 
+### Typography
+
+In version 4, `Typography` component has been removed, and there are changes on the API of [Paragraph](https://v4.f36.contentful.com/components/paragraph/), [Heading](https://v4.f36.contentful.com/components/heading/), [SectionHeading](https://v4.f36.contentful.com/components/section-heading/), [Subheading](https://v4.f36.contentful.com/components/subheading/), and [DisplayText](https://v4.f36.contentful.com/components/display-text/) to align with our code [style guide](https://github.com/contentful/forma-36/blob/forma-v4/docs/code-style-guide.md).
+
+#### How to migrate your Typography components
+
+To migrate your `Typography` components to v4, run the following [codemod](https://github.com/contentful/forma-36/tree/forma-v4/packages/forma-36-codemod):
+
+`npx @contentful/f36-codemod`
+
+If you want to do it manually, you must transform your existing code as follows:
+
+```tsx
+import {
```

Comment:
Maybe we could split examples into smaller chunks? I guess it would be easier to read. 

1. Just components, without `Typography` wrapper - before and after examples
2. Components wrapped in the `Typography` - before and after examples
3. Last one with bunch of divs inside - before and after examples

WDYT?